modules/charon-core/src/main/java/org/wso2/charon3/core/schema/SCIMConstants.java

```diff
@@ -66,6 +66,7 @@
     public static class CommonSchemaConstants {
 
         public static final String SCHEMAS = "schemas";
+        public static final String SCHEMAS_URI = "urn:ietf:params:scim:schemas:core:2.0:schemas";
```

Comment:
@MelinaGirl I guess the correct URI should be `urn:ietf:params:scim:schemas:core:2.0:Schema`?

Reply:
@ashensw No, in this case the URI is meant as a URI for the AttributeSchema "SCHEMAS", while your suggestion is refering to the URI for the schema resource (Specified in RFC7643 Section 7). Here it is for the attribute called "schemas", which is required for every resource to have (Specified in RFC7643 Section 3). Currently this required attribute is not handled the way all other attributes are handled, but by a separate "schemasList" attribute in the class AbstractSCIMObject, which causes inconsistencies.